pwiz_tools/Skyline/TestPerf/SmallMolLibrariesTutorialTest.cs

```diff
@@ -0,0 +1,431 @@
+                SkylineWindow.SetDisplayTypeChrom(DisplayTypeChrom.all);
+                //   �	From the View menu, choose Arrange Graphs and click Tiled (Ctrl+T).
+                SkylineWindow.ArrangeGraphs(DisplayGraphsType.Row); // With just two panes, Tiled may go either row or column, force row 
+                //   �	Right-click in a chromatogram graph and click Synchronize Zooming(leave if already checked).
+                SkylineWindow.SynchronizeZooming(true);
+                // �	Right-click in a chromatogram graph and click Legend to hide the legend.
+                SkylineWindow.ShowChromatogramLegends(false);
+                //   �	Right-click in a chromatogram graph, choose Transitions and click Split Graph (leave if already checked).
+                SkylineWindow.ShowSplitChromatogramGraph(true);
+            });
+
+            //  �	Select the molecule PC(16:0_18:1) and your spectra should appear as: 
+            FindNode("PC(16:0_18:1)");
+            PauseForScreenShot("Chromatograms", 13);
+
+            //Since there are only 38 precursors in this document, you may want to review all 38 to get an overall feel for how the XIC look prior to IMS filtering.Before starting this review, do the following:
+            //   �	From the View menu, click Library Match (Alt+1). Alternatively, click the library match ID marker displayed on the chromatogram.This should bring up the Library Match view:
+            //   �	Attach the Library Match view to the right edge of the Skyline window by clicking in the title bar and dragging until the mouse cursor is inside the right-side docking icon.
+            //   �	On the View menu, choose Retention Times and click Replicate Comparison (F8).
+            //   �	Attach the Retention Times view to the left of the Library Match view by clicking in the title bar and dragging until the mouse cursor is inside the left-side docking icon.
+            RunUI(() =>
+            {
+                SkylineWindow.ShowGraphRetentionTime(true, GraphTypeSummary.replicate);
+                //   �	Right-click in the Retention Times view and click Legend to hide the legend in this graph.
+                SkylineWindow.ShowRTLegend(false);
+            });
+            PauseForScreenShot("Library match", 14);
+
+            //   �	From the View menu, choose Peak Areas and click Replicate Comparison (F7).
+            RestoreViewOnScreen(15);
+
+            //   �	Attach the Peak Areas view above the Retention Times view by clicking in the title bar and dragging until the mouse cursor is inside the up-side docking icon.
+            //   �	Right-click in the Peak Areas view and click Legend to hide the legend in its graph.
+            RunUI( () => SkylineWindow.ShowPeakAreaLegend(false));
+            //    You should end up with a similar layout to that below:
+            RunUI(() => SkylineWindow.AutoZoomBestPeak());
+            WaitForGraphs();
+            PauseForScreenShot("Layout", 15);
```

Comment:
This isn't really a description of what I am taking the screenshot of. Is it the main window again?